sympy/solvers/ode/ode.py

```diff
@@ -731,6 +731,216 @@ def _helper_simplify(eq, hint, match, simplify=True, ics=None, **kwargs):
             rv = rv1
     return rv
 
+def dsubs(eq, trans):
+    r"""
+    Function that substitutes for variables in an Ordinary Differential Equation.
+
+    The inputs are -
+
+    ``eq``: The differential equation which is to be transformed.
+
+    ``trans``: It is a tuple of 2 elements containing -
```

Comment:
I would split this as separate arguments to the function.

Reply:
For now I've just made it a single arg and removed the reverse transformation. The reverse substitution can now be achieved by passing it as the same argument.